src/ResultsActivity/elements/PayOptions/PayOptions.js

```diff
@@ -79,17 +78,18 @@ function PayOptions({ offenderDetails, updateOffenderAttendance, date, openModal
           </Option>
         </Fragment>
       )}
-
       <Option data-qa="other-option" className="row-gutters">
-        {showRadioButton && (
-          <Radio name={offenderNo} onChange={renderForm} value="other" checked={selectedOption === 'other'}>
-            <VisuallyHidden>Other</VisuallyHidden>
-          </Radio>
-        )}
+        {showRadioButton &&
+          !absentReason && (
```

Comment:
Should `!absentReason` be covered by `showRadioButton`?

Reply:
I like it!